homeassistant/helpers/json.py

```diff
@@ -53,6 +53,8 @@ def json_encoder_default(obj: Any) -> Any:
         return obj.as_dict()
     if isinstance(obj, Path):
         return obj.as_posix()
+    if isinstance(obj, datetime.datetime):
```

Comment:
This one was missed from https://github.com/home-assistant/core/blob/eb56c7e1b74abfebf70b9b13ad7579b76e6f7783/homeassistant/helpers/json.py#L33